src/Apcu/bootstrap.php

```diff
@@ -13,18 +13,61 @@
     return;
 }
 
+if (extension_loaded('Zend Data Cache')) {
+    // Zend Data Cache refuses to load in the CLI, regardless of the configuration
+    if ('phpdbg' !== PHP_SAPI && 'cli' !== PHP_SAPI && false !== ini_get('zend_datacache.apc_compatibility')) {
+        require_once __DIR__.DIRECTORY_SEPARATOR.'bootstrap_zend.php';
+    }
+}
+
 if (!function_exists('apcu_add')) {
-    function apcu_add($key, $var = null, $ttl = 0) { return apc_add($key, $var, $ttl); }
```

Comment:
please keep these on a single line as they were, that'll help review, and as a change is unrelated to what you want to achieve here